src/test/ui-fulldeps/feature-gate-plugin.stderr

```diff
@@ -15,6 +15,6 @@ LL | #![plugin(empty_plugin)]
    |
    = note: `#[warn(deprecated)]` on by default
 
-error: aborting due to previous error
+error: aborting due to previous error, one warning emitted
```

Comment:
It seems better just to stick with the usual warning, rather than merge it into the error message. The compilation isn't aborting due to the warning at all.

Reply:
True, but that unnecessary wastes vertical space, which I. keep hearing complaints about.

Reply:
Maybe we could have `error: aborting due to previous error (1 warning emitted)` or `error: aborting due to previous error; 1 warning emitted` instead, then. A comma seems inappropriate here.